TezosKit.podspec

```diff
@@ -17,7 +17,8 @@ Pod::Spec.new do |s|
 
   s.dependency "BigInt", "~> 3.1"		
   s.dependency "MnemonicKit"
-  s.dependency "Sodium", "~> 0.7.0"  
+  s.dependency "PromiseKit"
```

Comment:
I would reconsider adding any dependencies to the library, especially one like PromiseKit.  The common pattern is to make subspecs that update your public API's to be friendly with other libraries.  In this case you could have a TezosKit/PromiseKit subspec that adds additional promisekit friendly API's to the standard ones. You could also create a TezosKit/Reactive to make an RXSwift friendly API.